src/utils/constants.js

```diff
@@ -63,7 +64,464 @@ export const subkindIcons = {
 }
 
 export const buttonCatalog = {
-  'E-mail': {
+  'Make a button': {
+    'open-app': {
+      kind: 'application',
+      is_primary: true,
+      configuration: {
+        label: 'To open an Application',
+        description: 'Allows to create a button that opens an Application.',
+        default: '',
+        color: colors.blue,
+        image_url: 'question_solid'
+      }
+    },
+    'open-link': {
+      kind: 'link',
+      is_primary: true,
+      configuration: {
+        label: 'To open Web Page',
+        description: 'Allows to create a button that opens a Web Page.',
+        color: colors.blue,
```

Comment:
There is a default color in the system, so I think we should remove all these `color: colors.blue`. This will allow us centrally to change the colors of all the buttons that haven't got a specific color set.

Reply:
Right, but I added them because I couldn't select a different color when editing the buttons' details. So yes, you're right, I'll take a look at it.